platform-common/src/main/java/com/softicar/platform/common/core/singleton/CurrentSingletonSet.java

```diff
@@ -37,4 +37,12 @@ public static void set(SingletonSet singletonSet) {
 
 		INSTANCE.set(singletonSet);
 	}
+
+	/**
+	 * Removes the current {@link SingletonSet}.
+	 */
+	public static void reset() {
```

Comment:
So, shouldn't this method be called `remove`, or `clear`?

Reply:
renamed to `remove`